app/models/article.rb

```diff
@@ -0,0 +1,8 @@
+class Article < ApplicationRecord
+    has_one :category
```

Comment:
I feel like this should be belongs_to as an article belongs to a category instead of having a category.